pkg/private/tar/tar.bzl

```diff
@@ -40,9 +41,12 @@ SUPPORTED_TAR_COMPRESSIONS = (
 _DEFAULT_MTIME = -1
 _stamp_condition = Label("//pkg/private:private_stamp_detect")
 
-def _remap(remap_paths, path):
+def _remap(ctx, path):
```

Comment:
Please revert this change and pass the `remap_paths` to this function. We should try to limit the number of locations where `ctx` is passed.